spec/models/latest_blood_pressures_per_patient_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe LatestBloodPressuresPerPatient, type: :model do
   describe 'Associations' do
-    it { should belong_to(:facility) }
+    it { should belong_to(:bp_facility) }
```

Comment:
Let's use the `it { is_expected.to belong_to }` syntax [[ref](http://www.betterspecs.org/#expect)]